src/upgrades/pre_0_14_0/cas_migration.rs

```diff
@@ -147,12 +147,33 @@ impl UpgradeAggregateStorePre0_14 for CasMigration {
                     UnconvertedEffect::Success { events } => {
                         let mut full_events: Vec<CertAuthEvent> = vec![]; // We just had numbers, we need to include the full events
                         for old_event in events {
-                            full_events.push(old_event.into());
+                            match old_event {
+                                Pre0_14_0CertAuthEvent::AspaConfigAdded { .. }
+                                | Pre0_14_0CertAuthEvent::AspaConfigRemoved { .. }
+                                | Pre0_14_0CertAuthEvent::AspaConfigUpdated { .. }
+                                | Pre0_14_0CertAuthEvent::AspaObjectsUpdated { .. } => {
+                                    // we only expect AspaObjectsUpdated to be possible outside of
+                                    // Aspa related commands, e.g. because of a key rollover, but
+                                    // to be sure.. we do not migrate any of the ASPA events in
```

Comment:
When you say "we do not migrate" do you mean "we keep without modifiction" ?

Reply:
I mean that these events don't need to be kept. We track the ASPA stuff separately and add back ASPA configs in the end - and re-issue ASPA objects then.